### ISSUE TITLE
When depositing, error: All decryptable notes should be in the noteToNullifier map

### ISSUE THREAD

**Issue by gjfrn** [iron-fish/ironfish#1440]
### What happened?

When attempting to deposit using yarn start deposit, I get the following error:

All decryptable notes should be in the noteToNullifier map
![Screenshot 2022-05-18 at 13 46 00](https://user-images.githubusercontent.com/95665400/169041820-58191e7e-0a31-45c6-877d-d94226120113.png)

### Version

0.1.34

### Debug output

```shell
yarn run v1.22.17
$ yarn build && yarn start:js debug
$ tsc -b
$ cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node bin/run debug
Database in use, skipping output that requires database.
Stop the node and run the debug command again to show full output.

Iron Fish version       0.1.34 @ src
Iron Fish library       0.0.12 @ src
Operating system        Darwin arm64
CPU model(s)            Apple M1
CPU threads             8
RAM total               16.00 GiB
Node version            v16.5.0
ironfish in PATH        true
Telemetry enabled       true
✨  Done in 2.25s
```

### Relevant log output

```shell
yarn run v1.22.17
$ yarn build && yarn start:js deposit
$ tsc -b
$ cross-env OCLIF_TS_NODE=0 IRONFISH_DEBUG=1 node bin/run deposit
All decryptable notes should be in the noteToNullifier map
✨  Done in 16.22s.
```

### Graffiti

GFIRONFISH

**Comment by rohanjadvani** [comment]
You can run `ironfish accounts:rescan --reset` to fix this bug. We're currently investigating the root cause of this.

**Comment by lwisne** [comment]
Thanks for the offer - if you still have this log, send it along!

If not, no problem - it took us far longer than it should have to respond.

**Comment by hughy** [comment]
Errors like this one should be resolved by the Wallet 2.0 refactor released in [v0.1.48](https://github.com/iron-fish/ironfish/releases/tag/v0.1.48)

Please open a new issue if you run into this again!
